src/guide/typescript-support.md

```diff
@@ -0,0 +1,280 @@
+# TypeScript Support
+
+> [Vue CLI](https://cli.vuejs.org) provides built-in TypeScript tooling support.
+
+## Official Declaration in NPM Packages
+
+A static type system can help prevent many potential runtime errors, especially as applications grow. That's why Vue is written in TypeScript. This means you don't need any additional tooling to use TypeScript with Vue - it has a first-class citizen support
```

Comment:
Second sentence is a fragment, how about:

>A static type system can help prevent many potential runtime errors as applications grow, which is why Vue 3 is written in TypeScript.

Reply:
Also missing a period at the end of the paragraph

Reply:
Fixed! Thank you @sdras 🙇🏻‍♀️